BrowserKit/Sources/WebEngine/WKWebview/Scripts/ReaderMode/WKReaderModeHandlers.swift

```diff
@@ -0,0 +1,160 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+import GCDWebServers
+
+protocol WKReaderModeHandlersProtocol {
+    func register(_ webServer: WKEngineWebServerProtocol,
+                  readerModeConfiguration: ReaderModeConfiguration)
+}
+
+public struct ReaderModeConfiguration {
+    var loadingText: String
+    var loadingFailedText: String
+    var loadOriginalText: String
+    var readerModeErrorText: String
+    var cachedReaderModeStyle: [String: Any]?
+}
+
+struct WKReaderModeHandlers: WKReaderModeHandlersProtocol {
+    private let ReaderModeStyleHash = "sha256-L2W8+0446ay9/L1oMrgucknQXag570zwgQrHwE68qbQ="
```

Comment:
Just curious about how we use the hash value in the response. If you know the answer

Reply:
It seems it's used for the [`HIGHLIGHT_CSS` in `FindInPage.js`](https://github.com/mozilla-mobile/firefox-ios/blob/2cb9cf505f353a8c2da67ac67b4bcef73b357d97/focus-ios/Blockzilla/FindInPage.js#L39-L54). That might mean we can remove it since `FindInPage.js` won't be used anymore 🤔

Reply:
I'll create a task so we don't forget about it later on